controllers/expense.controller.js

```diff
@@ -7,4 +7,23 @@ exports.get = async (req, res) => {
     catch (err) {
       res.status(422).send({answer: err});
     }
+  }
+
+  exports.add = async (req, res) => {
+    const { shop } = req.body;
+    const { cost } = req.body;
+    if (!shop || !cost) {
+      res.status(422);
+      return res.send({answer: "Shop name or price input is not defined."});
+    }  
+    if (cost < 0 && isNaN(cost)) {
```

Comment:
check with OR logic operator, because if it's nan how are going to check if it's less than 0